features/topics.yml

```diff
@@ -1,6 +1,12 @@
 name: Topics
-description: The Topics API reports a user's likely subject-matter interests (as determined by the browser, known as topics) to iframes. This allows for interest-based advertising that does not rely on user's exact browsing history, third-party cookies, or fingerprinting.
+description: The topics API reports a user's likely subject-matter interests (as determined by the browser, known as topics) to iframes. This allows for interest-based advertising that does not rely on user's exact browsing history, third-party cookies, or fingerprinting.
 spec: https://patcg-individual-drafts.github.io/topics/
+discouraged:
+  removal_date: 2026-06-01 # Chrome 150
```

Comment:
https://chromestatus.com/feature/5135370673061888 mentions 144, not 150. Where does version 150 come from?